tests/test_TFNativeOp.py

```diff
@@ -3033,6 +3033,95 @@ def test_blocksparse_simple_feature_axis1():
   assert_allclose(result, y_test)
 
 
+def _run_rnnt(acts, labels, input_lengths, label_lengths,
+              expected_costs, expected_grads, blank):
```

Comment:
It would be nice if we can also have a reference implementation. E.g. in pure TF, with `tf.while_loop` over `T+U+1`, how you manually would write down the forward computation (and backward is automatically via backprop), also in log space. But not needed for this initial pull request.